modules/rtm/R/invert.auto.R

```diff
@@ -67,11 +67,30 @@ invert.auto <- function(observed, invert.options, return.samples=TRUE, save.samp
     i.try <- 1
     while(try.again & i.try <= n.tries){
```

Comment:
@bpbond Because `i.try` is set to 1 right before this, the only way that could happen is if the user manually passed `n.tries = 0`, which is a nonsensical setting. But in that case, the `while` loop would not do anything to begin with.